great_expectations/core/batch.py

```diff
@@ -304,7 +308,7 @@ def __init__(
         )
 
     @staticmethod
-    def _validate_batch_request(
+    def _validate_init_parameters(
```

Comment:
Same comment as earlier: Do we really want to rename _validate_batch_request() into _validate_init_parameters()?.. Happy to discuss.